maxim/tests/test_langgraph.py

```
@@ -158,10 +152,13 @@ async def ask_agent(query: str) -> str:
 @flask_app.post("/chat")
 @trace(logger=logger, name="chat_v1", id=lambda: request.headers["reqId"])
 async def handle():
-    current_trace().add_tag("test", "yes")
-    resp = await ask_agent(request.json["query"])
+    trace = current_trace()
+    if trace is None:
+        raise ValueError("current_trace is None")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)